### ISSUE TITLE
CI fails on rocksdb

### ISSUE THREAD

**Issue by ctb** [sourmash-bio/pyo3_branchwater#79]
over in https://github.com/sourmash-bio/pyo3_branchwater/pull/58, we get:
```
 error: failed to run custom build command for `librocksdb-sys v0.11.0+8.1.1`

Caused by:
  process didn't exit successfully: `/home/runner/work/pyo3_branchwater/pyo3_branchwater/target/release/build/librocksdb-sys-07a244c34ecb3d24/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at 'Unable to find libclang: "couldn't find any valid shared libraries matching: ['libclang.so', 'libclang-*.so', 'libclang.so.*', 
```

I think this can be dealt with by fixing https://github.com/sourmash-bio/pyo3_branchwater/issues/20, where we will (probably) switch to running stuff inside of a conda environment.

**Comment by ctb** [comment]
#80 fixes this by (I think) installing the full compilers suite, as well as using a more recent version of rust. I just deleted the maturin CI for now.

**Comment by ctb** [comment]
(but leaving this open as a reminder ;))

**Comment by ctb** [comment]
Fixed in #58/#80.
